src/components/Login/Panels/LoginPanel.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Auth,
+  AuthProvider,
+  FacebookAuthProvider,
+  GoogleAuthProvider,
+  OAuthProvider,
+  User
+} from 'firebase/auth'
+import { useSetAtom } from 'jotai'
+
+import { panelAtom } from '@/atoms/login'
+import LoginButton from '@/components/Login/LoginButton'
+import Logo from '@/components/UI/logoSvg'
+import { useAuth } from '@/context/Firebase/Auth/context'
+
+export interface LoginPanelInterface {
+  linkAccount: boolean // if true, will link the account to the logged in user, if false, will sign in with the provider
+  displayGoogle: boolean
+  displayFacebook: boolean
+  displayMicrosoft: boolean
+  displayPhone: boolean
+}
+
+const LoginPanel = ({
+  linkAccount,
+  displayGoogle,
+  displayFacebook,
+  displayMicrosoft,
+  displayPhone
+}: LoginPanelInterface) => {
+  const { externalAuthSignIn, linkAuthProvider, currentUser, auth } = useAuth()
+  const setPanel = useSetAtom(panelAtom)
+
+  const googleIcon = <Logo name='Google' viewBox='0 0 24 24' />
+  const facebookIcon = <Logo name='Facebook' viewBox='0 0 24 24' />
+  const microsoftIcon = <Logo name='Microsoft' viewBox='0 0 24 24' />
+  const phoneIcon = <Logo name='Phone' viewBox='0 0 24 24' />
+
+  const googleProvider = new GoogleAuthProvider()
+  const facebookProvider = new FacebookAuthProvider()
+  const microsoftProvider = new OAuthProvider('microsoft.com')
+
+  function buttonString(providerString: string) {
+    return !linkAccount
+      ? 'Continue with ' + providerString
+      : 'Link ' + providerString + ' Account'
+  }
+
+  function handleExternalAuth(
+    auth: Auth,
+    currentUser: User | null,
+    provider: AuthProvider
+  ) {
+    if (!linkAccount) {
+      externalAuthSignIn?.(auth, provider)
+    } else {
+      if (currentUser !== null) {
+        linkAuthProvider?.(currentUser, provider)
+      }
+    }
+  }
+
+  return (
+    <>
+      <div className='text-x1 mx-auto text-center font-sans'>
+        Use any one of your profiles
```

Comment:
Probably not needed to have this text here as it's pretty self explanatory what to do.